src/unleash-proxy.ts

```diff
@@ -127,10 +127,12 @@ export default class UnleashProxy {
                     ...standardResponses(401, 500, 501, 503),
                     200: featuresResponse,
                 },
-                description:
-                    'This endpoint returns the list of feature toggles that the proxy evaluates to enabled and disabled for the given context. As such, this endpoint always returns all feature toggles the proxy retrieves from unleash. Useful if you are migrating to unleash and need to know if the feature flag exists on the unleash server. However, using this endpoint will increase the payload size transmitted to your applications. Context values are provided as query parameters.',
-                summary:
-                    'Retrieve enabled feature toggles for the provided context.',
+                description: `This endpoint returns the list of feature toggles that the proxy evaluates to enabled and disabled for the given context. As such, this endpoint always returns all feature toggles the proxy retrieves from Unleash.
```

Comment:
```suggestion
                description: `This endpoint returns the list of feature toggles, and whether they are enabled or disabled for the given context. As such, this endpoint always returns all feature toggles the proxy retrieves from Unleash (in contrast to the `/proxy` endpoints that only return enabled toggles).
```

Reply:
Yeah, I like this! I'll make the changes in a new PR.